### PR TITLE
Add immortality and tests

### PR DESCRIPTION
**Pull request by joelamouche**
- new 'immortality' option allows to create tx that dont expire

### CODE REVIEW

File: src/commands/createAndSendCommand.ts [hunk 1]
```diff
@@ -39,11 +39,16 @@ export const createAndSendTxCommand = {
         describe: "activates sudo mode",
         type: "boolean",
         default: false,
+      })
+      .positional("immortality", {
```
Comment (crystalin) [inline src/commands/createAndSendCommand.ts:43]: BTW, any reason why all the parameters are positional ? I always find it less flexible that way

File: src/methods/createAndSendTx.ts [hunk 2]
```diff
@@ -50,7 +50,8 @@ export async function createAndSendTx(
       });
     },
   };
-  await txExtrinsic.signAndSend(address, { signer });
+  let options=txArgs.immortality?{ signer,era:0 }:{ signer }
```
Comment (crystalin) [inline src/methods/createAndSendTx.ts:53]: We should probably start to enforce formatting on that repo

**Review by crystalin — APPROVED** [review]
All good 👍

File: src/commands/createAndSendCommand.ts (hunk 1 above)
Comment (joelamouche) [reply src/commands/createAndSendCommand.ts:43]: As opposed to making them optional you mean?

File: src/methods/createAndSendTx.ts (hunk 2 above)
Comment (joelamouche) [reply src/methods/createAndSendTx.ts]: done

File: src/commands/createAndSendCommand.ts (hunk 1 above)
Comment (joelamouche) [reply src/commands/createAndSendCommand.ts:43]: @crystalin ?

Comment (joelamouche) [reply src/commands/createAndSendCommand.ts:43]: please point me to the right feature because I'm not aware of any other way to do that

**Pull request merged**